mdn/developer-portal
# 1233 post review fixups

This changeset includes a number of small fixups (mainly CSS) which were found either via code review of earlier PRs or from my cross-browser testing
## Key changes:

- Person page when they feature topics (eg see Jen Simmonds on Dev)
  - [ ] short title (eg "CSS") was too short to force reflow of the description line. Fixed now.
  - [ ] glitched/breaking responsive layout in narrower viewport widths - fixed now


- [ ] Topic Cards (on homepage and on Person page) use the `card_description` field, not the main `description` field, fixing a regression compared to production/old design
 
- [ ] HGeader: Mozilla Developer logo shifted 1x higher to match designs, now sits on same baseline as the menu text
- [ ] Events list: the optional rich-text area/CTA button (the one we use to link to Sponsorship page) now has even, less gappy, spacing above and below

- [ ] Filter list on Events and/or People, when multiple critera are being filtered by used to suffer from poor spacing before/after the "Cancel X" links - this has been fixed

- [ ] Mobile view of homepage: The CTA bar (the darker gray one above the People section on the homepage) needed the spacing adjusting slightly

- [ ] Filter form (on Posts and Events) needed button styling fixing on iOS Safari to avoid rounded corners


For the full hit-list of items addressed, please see the main issue #1233 

## How to test

- the code is staged on Dev. I would drive around to view each of the items mentioned above and confirm they look OK. Because they've been cross-browser tested already (and to save time/minimise review workload), doing this in any single browser will be enough for now, I thikn
